src/common/modals/ChangeLogs.js

```diff
@@ -88,7 +88,7 @@ const ChangeLogs = () => {
 
 export default memo(ChangeLogs);
 
-const Container = styled.div`
+export const Container = styled.div`
```

Comment:
Don't export components between components, just copy it over or extract it to a common file